src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, AreaCalculator {
+    protected String name;
```

Comment:
There is no need to name field, Name could be taken using Object#getClass

Reply:
I changed it to private.